src/test/java/seedu/address/logic/commands/ClearPropertyCommandTest.java

```diff
@@ -5,28 +5,28 @@
 
 import org.junit.jupiter.api.Test;
 
-import seedu.address.model.AddressBook;
 import seedu.address.model.Model;
 import seedu.address.model.ModelManager;
+import seedu.address.model.PropertyBook;
 import seedu.address.model.UserPrefs;
 
-public class ClearCommandTest {
-
+class ClearPropertyCommandTest {
     @Test
-    public void execute_emptyAddressBook_success() {
+    public void execute_emptyPropertyBook_success() {
         Model model = new ModelManager();
         Model expectedModel = new ModelManager();
 
-        assertCommandSuccess(new ClearCommand(), model, ClearCommand.MESSAGE_SUCCESS, expectedModel);
+        assertCommandSuccess(new ClearPropertyCommand(), model, ClearPropertyCommand.MESSAGE_SUCCESS,
+                expectedModel);
     }
 
     @Test
-    public void execute_nonEmptyAddressBook_success() {
+    public void execute_nonEmptyPropertyBook_success() {
         Model model = new ModelManager(getTypicalAddressBook(), new UserPrefs());
         Model expectedModel = new ModelManager(getTypicalAddressBook(), new UserPrefs());
```

Comment:
Rmb to update this to `getTypicalPropertyBook()`

Reply:
Fixed